lib/msal-common/test/client/RefreshTokenClient.spec.ts

```diff
@@ -52,9 +52,9 @@ describe("RefreshTokenClient unit tests", () => {
         const authResult = await client.acquireToken(refreshTokenRequest);
 
         expect(JSON.parse(authResult)).to.deep.eq(AUTHENTICATION_RESULT.body);
-        expect(createTokenRequestBodySpy.calledWith(refreshTokenRequest)).to.be.ok;
+        expect(createTokenRequestBodySpy.calledWith(refreshTokenRequest)).to.be.true;
 
-        expect(createTokenRequestBodySpy.returnValues[0]).to.contain(`${AADServerParamKeys.SCOPE}=${TEST_CONFIG.DEFAULT_GRAPH_SCOPE}%20${Constants.OPENID_SCOPE}%20${Constants.PROFILE_SCOPE}%20${Constants.OFFLINE_ACCESS_SCOPE}`);
+        expect(createTokenRequestBodySpy.returnValues[0]).to.contain(`${AADServerParamKeys.SCOPE}=${TEST_CONFIG.DEFAULT_GRAPH_SCOPE}%20${Constants.OFFLINE_ACCESS_SCOPE}`);
```

Comment:
RefreshToken client should return token with all three default scopes. We should why that's not the case when we update tests.